minfraud/request.py

```diff
@@ -91,29 +282,43 @@ def maybe_hash_email(transaction):
     if address is None:
         return
 
-    address = address.lower().strip()
-
-    at_idx = address.rfind("@")
-    if at_idx == -1:
+    address = _clean_email(address)
+    if not address:
```

Comment:
How about `if address is None`?

Reply:
I was thinking the address could be an empty string.

Reply:
Can it? Won't this function always return `None` in that case?

Reply:
It looks like no, updated the check.